CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,51 @@
+# Contributor Covenant Code of Conduct
+
+## Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as
```

Comment:
please also add a landing page

Reply:
What do you mean?

Reply:
I can't push forward with this PR without a proper landing page in this project :(

Reply:
Why?

Reply:
How can one know how to conduct if one cannot land?

Reply:
I understand, I will contact GitHub to schedule a session for you. Thank you for coming forward and sharing.

Reply:
Please do!
While you are at it, please ask them to review the COC that you wrote for me as well

Thanks !